### PR TITLE
Dummy aws credentials during setup.

### PR DESCRIPTION
**Pull request by tomeksabala**
During setup user will be prompted if he/she wishes to have dummy aws credentials setup if no credentials found in HOME directory.

### CODE REVIEW

File: util/__init__.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 # Some settings
 # If you run docker with a sudo, set this envvar to the sudo command: 'sudo'
+from util.dummy_aws_credentials import create_dummy_aws_config
```
Comment (jonathansberry) [inline util/__init__.py]: This should be moved up to line 6 above the comments  about SUDO user option. KEep the comments next to the right piece of code.

File: util/__init__.py [hunk 2]
```diff
@@ -173,6 +175,7 @@ def setup(args, extra):
     print('Setting up the Meerkat codebase...')
     print('This will destroy changes, resetting everything to the remote.')
 
+    create_dummy_aws_config()
```
Comment (jonathansberry) [inline util/__init__.py]: This line should probably be after checking with the user whether they want to continue. If the user says no, we should probably not do anything at all. Maybe just move it down two lines?

File: util/dummy_aws_credentials.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    if not os.path.exists(config_directory):
+        if raw_input(confirmation_message.format(config_directory)) not in ['n', 'N', 'No', 'no', 'NO']:
+            os.makedirs(config_directory)
+            with file('{}/config'.format(config_directory), mode='w') as config_file:
+                config_file.write(aws_config)
+            with file('{}/credentials'.format(config_directory), mode='w') as credentials_file:
+                credentials_file.write(aws_credentials)
+
+
+if __name__ == '__main__':
```
Comment (jonathansberry) [inline util/dummy_aws_credentials.py]: :+1:

File: util/dummy_aws_credentials.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python2
+
+import os
+from os.path import expanduser
+
+aws_config = """[default]
+output = json
+region = eu-west-1
+"""
+
```
Comment (jonathansberry) [inline util/dummy_aws_credentials.py]: Random blank line?

**Comment by tomeksabala** [comment]
@jonathansberry Thanks for valid suggestions. I've included all of them.

**Pull request merged**